_projects/nested_languages.md

```diff
@@ -1,7 +1,8 @@
 ---
 collaborating_projects:
   - coala
-desc: "Multiple programming languages can coexist in the same source file - coala should support writing code analysis that only works on parts of files."
+desc: "Multiple programming languages can coexist in the same source file - coala should support writing code analysis
```

Comment:
break this line at 80cpl, and align indented text on next line under first letter of this line.